src/components/Cards/item.tsx

```diff
@@ -31,17 +32,14 @@ const Item = ({
       sx={{
         display: 'flex',
         minHeight: '100px',
-        width: '348px',
+        width: 'auto',
         flex: { md: masonry ? '0 0 auto' : '1 1 0' },
+        boxShadow: '0 8px 38px 7px #002b551a',
```

Comment:
is it possible not to hardcode values like `#002b551a`?

Reply:
for this one, I'll remove it and then we'll check the shadow from B2BP where we'll use it instead of hardcoding it